packages/sdks-tests/src/e2e-tests/countdown.spec.ts

```diff
@@ -0,0 +1,24 @@
+import { expect } from '@playwright/test';
+import { test } from '../helpers/index.js';
+
+test.describe('Symbol with JS Code', () => {
+  test('correctly updates countdown date', async ({ page, sdk }) => {
+    test.fail(
+      sdk === 'qwik' || sdk === 'react' || sdk === 'rsc',
```

Comment:
curious on why this doesn't work in the react SDK, `props.symbol.data` should update in this case causing a re-render right?

Reply:
yeah, that's the bug we need to solve. I couldn't fix it quickly enough but wanted to commit the test for future. 

Most surprising is that this works in react native SDK. I can't see how it handles bindings any differently than react SDK.